tests/multiprocessing_sync_tests.py

```diff
@@ -3,57 +3,77 @@
 
 from syllabus.tests import SyncTestCurriculum
 from syllabus.core import make_multiprocessing_curriculum, make_ray_curriculum
-from syllabus.tests import test_single_process, test_native_multiprocess, test_ray_multiprocess, create_synctest_env
+from syllabus.tests import create_synctest_env
+from syllabus.tests import test_single_process as helper_single_process
+from syllabus.tests import test_native_multiprocess as helper_native_multiprocess
+from syllabus.tests import test_ray_multiprocess as helper_ray_multiprocess 
```

Comment:
Maybe we should just rename these functions at the source instead of importing them with new names here

Reply:
I think you can rename them to `run_single_process`, `run_native_multiprocess`, and `run_ray_multiprocess`

Reply:
As of now, I don't know if these functions are used in anywhere else apart from `tests/multiprocessing_sync_tests.py` or not. Renaming these at source may cause breaking the codes if these functions are being used in any other scripts. Before I rename these at source, I have to be very careful about their usage. For other comments, I agree with you.